ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/defs_node_type.schema.yml

```diff
@@ -946,6 +946,14 @@ $defs:
               enabled:
                 type: bool
                 default: false
+              mode:
+                type: str
+                valid_values:
+                  - boundary
+                  - transparent
+              boundary_one_step:
```

Comment:
please look at review in #3666 as I think the model should be different.

Reply:
Yes, updated